files/tagtacular/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "tagtacular",
+  "repo": "burnsbert/tagtacular",
+  "files": {
+    "include": ["tagtacular.js", "tagtacular.min.js", "tagtacular.css"]
```

Comment:
Sorry, I should have included the path on those files in update.json. I
think I corrected the issue in the commit I just made.

Eric

On Wed, Aug 12, 2015 at 2:24 PM, Dmitriy Akulov <notifications@github.com>
wrote:

> In files/tagtacular/update.json
> <https://github.com/jsdelivr/jsdelivr/pull/6374#discussion_r36895462>:
>
> > @@ -0,0 +1,8 @@
> > +{
> > +  "packageManager": "github",
> > +  "name": "tagtacular",
> > +  "repo": "burnsbert/tagtacular",
> > +  "files": {
> > +    "include": ["tagtacular.js", "tagtacular.min.js", "tagtacular.css"]
>
> These files dont exist here
> https://github.com/burnsbert/tagtacular/tree/1.0.1
> The bot will fail
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jsdelivr/jsdelivr/pull/6374/files#r36895462>.
>